utils/layout.py

```diff
@@ -3,7 +3,9 @@
 import torch
 
 
-def ch_locations_2d(dataset_name):
+def ch_locations_2d(args):
+    dataset_name, root_dir = args.dataset, args.root_dir
```

Comment:
Use Hydra util

Reply:
Like I note above, I get it on line 25 of train.py. Not quite sure what to do here. Would you like me to call `get_original_cwd()` in every function that needs to know the root dir?